cob_hardware_config/components/sick_s300/sick_s300.yaml

```diff
@@ -1,9 +1,4 @@
-port: /dev/ttyUSB0
 baud: 500000
 scan_duration: 0.025 #no info about that in SICK-docu, but 0.025 is believable and looks good in rviz
 scan_cycle_time: 0.040 #SICK-docu says S300 scans every 40ms
-inverted: false
-scan_id: 7
-frame_id: base_laser_rear_link
 publish_frequency: 12
-scan_intervals: [[-2.0, 2.0]] #[rad] these intervals are included to the scan
```

Comment:
It's not even just that, i.e. deciding whether it's "instance-general" (identical for all instances of the same component/module/robot) or "instance-specific" (possibly different for each instance of the same component/module/robot)...

...I had severe brain-knots when needing to decide whether to call s.th. an instance, a component, a module, a bundle....or simply keep it in robot...

Thus, so far, I don't see the real benefit of this restructuring myself...I know where we want to go, but I have no concept for it (yet)...
However, I think we should go for an approach where we combine/generalize things that are **currently** identical - even if at a future point we might introduce a new instance that requires different configuration....I think we should handle such case once it occurs - rather than already considering the most general, galactical configuration structure possible...it's all about making our life easier **atm**, i.e. **now**, i.e. with the requirements we **currently** have (not the ones we might have in the future - maybe)

If someone knows a good concept to do so, please provide input @ipa-nhg @ipa-mdl